regression/pages/studio/course_outline_page.py

```diff
@@ -0,0 +1,72 @@
+"""
+Course Outline Page for Studio
+"""
+
+from edxapp_acceptance.pages.studio.overview import CourseOutlinePage
+
+from regression.pages.studio.utils import \
+    click_confirmation_prompt_primary_button
+from regression.tests.helpers import get_url
+
+
+class CourseOutlinePageExtended(CourseOutlinePage):
+    """
+    Coure Outline Extended Page for Studio
+    """
+    @property
+    def url(self):
+        """
+        Construct a URL to the page within the course.
+        """
+        return get_url(self.url_path, self.course_info)
+
+    def add_section_with_name(self, text):
+        """
+        Adds Section clicking the (main) New Section button and given name
+        """
+        self.q(css='.wrapper-mast nav.nav-actions .button-new').click()
+        self.wait_for_ajax()
+        self.q(
+            css='.incontext-editor-input[value="Section"]'
+        ).results[0].send_keys(text)
+
+        self.q(css='.content').first.click()
+
+    def add_subsection_with_name(self, text):
+        """
+        Adds Subsection clicking the subsection button of a section
+        There should be one Section available
+        """
+        self.q(css='.button-new[data-default-name="Subsection"]').click()
+        self.wait_for_ajax()
+        self.q(
+            css='.incontext-editor-input[value="Subsection"]'
+        ).results[0].send_keys(text)
+
+        self.q(css='.content').first.click()
+
+    def grade_as_verification(self):
+        """
+        Returns:
+        List of grades available in Grade as drop down in subsection settings
+        """
+        return self.q(css='#grading_type option').text
+
+    def subsection_settings_cancel(self):
```

Comment:
similar here...i'd raname this slightly so it starts with the verb: `cancel_subsection_settings`